plugins/pulp_docker/plugins/distributors/rsync_distributor.py

```diff
@@ -0,0 +1,112 @@
+from gettext import gettext as _
+import logging
+
+from pulp.common.config import read_json_config
+from pulp.plugins.distributor import Distributor
+from pulp.plugins.rsync import configuration as rsync_config
+
+from pulp_docker.plugins.distributors.publish_steps import DockerRsyncPublisher
+
+TYPES = []
```

Comment:
it should contain all 4 types associate with docker repos. i am adding all to the list.